traverse_test.go

```diff
@@ -46,3 +47,262 @@ func TestNetworksWithInvalidSearchTree(t *testing.T) {
 	assert.NotNil(t, n.Err(), "no error received when traversing an broken search tree")
 	assert.Equal(t, n.Err().Error(), "invalid search tree at 128.128.128.128/32")
 }
+
+func TestNetworksWithinV4SearchInV4Db(t *testing.T) {
```

Comment:
These probably could have been a single table driven test given that the structure of each test is pretty much identical.

Reply:
I've cleaned these up.  The tests are now much more succinct.